authority/provisioner/oidc.go

```diff
@@ -195,6 +202,28 @@ func (o *OIDC) Init(config Config) (err error) {
 		return err
 	}
 
+	// Set the identity getter if it exists, otherwise use the default.
+	if config.GetIdentityFunc == nil {
+		o.getIdentityFunc = DefaultIdentityFunc
+	} else {
+		o.getIdentityFunc = config.GetIdentityFunc
+	}
```

Comment:
This and the getIdentityFunc are probably introduced by a bad merge, they are managed in the controller.